### DISCUSSION TITLE
Advanced pages example, with variables and AiScript

### DISCUSSION THREAD

**Discussion by tassoman** [misskey-dev/misskey#9175] [General]
Is there some example on the web, explaining how to interact with variables and aiscript during page publishing?

**Comment by KawaneRio** [comment]
<ruby>A<rp>(</rp><rt>As </rt><rp>)</rp>F<rp>(</rp><rt>Far </rt><rp>)</rp>A<rp>(</rp><rt>As </rt><rp>)</rp> I<rp>(</rp><rt>I</rt><rp>)</rp>K<rp>(</rp><rt>Know </rt><rp>)</rp></ruby>, https://github.com/syuilo/aiscript/blob/master/translations/en/docs/get-started.md （🇯🇵https://github.com/syuilo/aiscript/blob/master/docs/get-started.md ）and [AiScript (v0.11.1) Playground](https://syuilo.github.io/aiscript/) are the only places that has AiScript resources. 
On a side note, Syuilo has announced that Misskey Pages on Misskey v13 will no longer support AiScript, but instead, there will be an entire new place named Misskey Play for all AiScript stuff. Source🔗https://misskey.io/notes/98flq8clij

**Reply by tassoman** [reply]
Thanks for your complete answer
